service/matrix/matrix.go

```diff
@@ -0,0 +1,66 @@
+package matrix
```

Comment:
Hi @arnocornette !

Thanks a lot for your efforts and contribution! I had only some really minor comments, mainly stylish. Your implementation looks solid and in compliance with the repo's contribution guidelines, special thanks for that! 🙏 

To make this complete, could we add some documentation for the service as well? You can take a look at these examples. 
[example service README](https://github.com/nikoksr/notify/tree/main/service/whatsapp) & [example go doc](https://github.com/nikoksr/notify/blob/main/service/whatsapp/doc.go)

Reply:
Your quick response to those comments is much-appreciated @arnocornette ! It seems that this [commit](https://github.com/nikoksr/notify/pull/410/commits/57c959c11c7596d4496ef9d4ae2bf73f36f4f014) somehow reverted the service [name](https://github.com/nikoksr/notify/pull/410/commits/57c959c11c7596d4496ef9d4ae2bf73f36f4f014#diff-73ab59a205b5c2707ca30efcfa6c13c94dfd8106c2bc2f77d0f39ffc8a14792eR2) :smile: 
Let's fix that together with [this](https://github.com/nikoksr/notify/pull/410/commits/57c959c11c7596d4496ef9d4ae2bf73f36f4f014#diff-73ab59a205b5c2707ca30efcfa6c13c94dfd8106c2bc2f77d0f39ffc8a14792eR2) link to point to the new service instead and I think we're perfect to go! :+1:

Reply:
Hey @svaloumas not a problem! I think I fixed your last few remarks 😄 . Let me know if you find some more!